packages/calypso-e2e/src/lib/components/editor-toolbar-component.ts

```diff
@@ -16,7 +16,7 @@ const selectors = {
 	switchToDraftButton: `${ panel } button.editor-post-switch-to-draft`,
 
 	// Preview
-	previewButton: `${ panel } :text("View"):visible`,
+	previewButton: `${ panel } [aria-label="Preview"]:visible`,
```

Comment:
I think this is suitable for now, but I will want to look at whether we can use accessible locators (eg. `getByRole`) in the future.